src/index.ts

```diff
@@ -200,6 +211,21 @@ async function* readStream(reader: Stream): AsyncGenerator<JsonValue> {
 
 export type startBlock = () => bigint;
 
+interface RetryConfig {
+  maxAttempts: number;
+  baseDelay: number;
+  maxDelay: number;
+  delay: number;
```

Comment:
Perhaps I'm missing something in the TS code but I don't see how this field is used. Is it different than the delay function?

Reply:
Yeah, it is a bit confusing... Maybe it's worth a rename, it's the active delay amount.